.github/workflows/ci.yml

```diff
@@ -148,32 +148,8 @@ jobs :
           java-version : '11'
           check-latest : true
 
-      - name : Publish to Maven Local
-        run : ./gradlew clean publishToMavenLocal --no-build-cache --no-daemon --stacktrace --no-parallel
-
-  publish-snapshot :
-    runs-on : ubuntu-latest
-    if : github.repository == 'square/anvil' && github.ref == 'refs/heads/main'
-    timeout-minutes : 25
-    needs :
-      - test-ubuntu
-      - test-gradle-plugin
-      - gradle-wrapper-validation
-      - publish-maven-local
-
-    steps :
-      - uses : actions/checkout@v3
-      - uses : actions/setup-java@v3
-        with :
-          distribution : 'temurin'
-          java-version : '11'
-          check-latest : true
-
-      - name : Publish Snapshots 1.9
-        run : ./gradlew clean publish --no-build-cache --no-daemon --stacktrace --no-parallel
-        env :
-          ORG_GRADLE_PROJECT_mavenCentralUsername : ${{ secrets.SONATYPE_NEXUS_USERNAME }}
-          ORG_GRADLE_PROJECT_mavenCentralPassword : ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
+      - name: Publish to Maven Local
+        run: ./gradlew clean publishToMavenLocal --no-build-cache --no-daemon --stacktrace
```

Comment:
Why is `--no-parallel` removed?

Reply:
It hasn't been necessary since [0.25.0](https://github.com/vanniktech/gradle-maven-publish-plugin/blob/main/CHANGELOG.md#0250-2023-03-23) from March 2023.